MeshGeoToolsLib/convertMeshNodesToGeometry.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include "MeshGeoToolsLib/convertMeshNodesToGeometry.h"
+
+#include "MeshLib/Mesh.h"
+#include "MeshLib/Node.h"
+
+namespace MeshGeoToolsLib
+{
+void convertMeshNodesToGeometry(MeshLib::Mesh const& mesh,
```

Comment:
Yes, you are right. I would move the existing `MeshLib::convertMeshToGeo` here `MeshGeoToolsLib` since `GeoLib` data structures as well as `MeshLib` data structures are needed. But the moving should be a separate PR.